### PR TITLE
Add support for root distance and dispersion in NtpdCollector

### PR DESCRIPTION
**Pull request by jaingaurav**
These values aide in debugging ntp issues.

### CODE REVIEW

File: src/collectors/ntpd/ntpd.py [hunk 1]
```diff
@@ -117,6 +117,30 @@ def get_ntpdc_stats(self):
                 data['max_error'] = val
             elif key == 'estimated error':
                 data['est_error'] = val
+            elif key == 'status':
+                data['status'] = val
+
+        return data.items()
+
+    def get_ntpdc_sysinfo_output(self):
+        return self.run_command([self.config['ntpdc_bin'], '-c', 'sysinfo'])
```
Comment (mattrobenolt) [inline src/collectors/ntpd/ntpd.py]: I'm not super familiar, but is there any case where this command may not exist or could fail?

Comment (jaingaurav) [inline src/collectors/ntpd/ntpd.py]: It's entirely possible, though I think unlikely if someone enabled this collector. It this an issue?

Comment (jaingaurav) [inline src/collectors/ntpd/ntpd.py]: Also note that the command already being used by this collector for the `kerninfo`

Comment (mattrobenolt) [inline src/collectors/ntpd/ntpd.py]: Sorry, I meant, is there a case where `ntpdc -c kerninfo` may work, but `ntpdc -c sysinfo` will not exist?

Comment (jaingaurav) [inline src/collectors/ntpd/ntpd.py]: From http://archive.ntp.org/ntp4/ChangeLog-stable if you search for `sysinfo` it seems that `sysinfo` and `kerninfo` were added at the same time. So the answer I guess is no, they should both exist.

Comment (mattrobenolt) [inline src/collectors/ntpd/ntpd.py]: :+1:

**Pull request merged**